learning-bitmark/contributing-to-bitmark/bup.md

```diff
@@ -0,0 +1,14 @@
+## What is BUP?
+A Bitmark Upgrade Proposal (BUP) is a design document providing
+information to Bitmark community, or describing a new feature for
+Bitmark or its processes or environment. A BUP shall provide a
+concise technical specification of the feature and a rationale for the
+feature.
+
+## Where I can find the BUPs?
+We maintain an entity at Github only for BUP, we want to keep a clean 
+environment where developers will not have distraction with other
+repos, this is the reason why we do not maintain it under
+Bitmark documentation ![repository](https://github.com/bitmark-inc/docs).
+
```

Comment:
Not sure you need the additional explanation (this is the reason why we do not maintain it under
Bitmark documentation ![repository](https://github.com/bitmark-inc/docs).)